test/jasmine/tests/gl2d_click_test.js

```diff
@@ -0,0 +1,146 @@
+var Plotly = require('@lib/index');
+var Lib = require('@src/lib');
+
+var createGraphDiv = require('../assets/create_graph_div');
+var destroyGraphDiv = require('../assets/destroy_graph_div');
+var customMatchers = require('../assets/custom_matchers');
+
+// cartesian click events events use the hover data
+// from the mousemove events and then simulate
+// a click event on mouseup
+// var click = require('../assets/click');
+var hover = require('../assets/hover');
+
+describe('Test click interactions:', function() {
```

Comment:
Nice tests. But unfortunately, the CircleCI runner still can't boot up `gl` contexts.

![image](https://cloud.githubusercontent.com/assets/6675409/18958964/8347661a-8632-11e6-86d6-4604eedf647e.png)

Referencing https://github.com/plotly/plotly.js/issues/241

Previously, we ignored the gl test suites in the karma CI [config](https://github.com/plotly/plotly.js/blob/708e0ffd9ee827fd2640334a072f86605dcb2f5d/test/jasmine/karma.ciconf.js#L22-L27), but more recently, I've been using this [`hasWebGLSupport`](https://github.com/plotly/plotly.js/blob/master/test/jasmine/assets/has_webgl_support.js) util directly in the test suites. See example in the [mapbox test](https://github.com/plotly/plotly.js/blob/708e0ffd9ee827fd2640334a072f86605dcb2f5d/test/jasmine/tests/mapbox_test.js#L279) - which outputs this

![image](https://cloud.githubusercontent.com/assets/6675409/18959147/1f19f530-8633-11e6-9cdb-f9e243768371.png)

making it a little clearer that we're skipping a few tests on CI (I think).